source/includes/_endpoint-deck.md

```diff
@@ -280,9 +280,11 @@ display | object | Stores settings used to load the analysis
 
 #### POST
 
-To create a new slide, POST an analysis to the slides catalog. The payload, described in the [below](#analyses) section, should be wrapped as a shoji:entity.
+To create a new slide, POST an analysis to the slides catalog. The payload,
+described in the [below](#analyses) section, should be wrapped as a shoji:entity.
 
-On success, the server returns a 201 response with a Location header containing the URL of the newly created slide entity.
+On success, the server returns a 201 response with a Location header containing
+the URL of the newly created slide entity with its first analysis.
```

Comment:
If I want to create a slide with 4 analyses on it, how do I do that? POST the first one to the deck to create a slide, GET that entity, then make 3 subsequent POSTs to its analyses catalog to add the others?

Or can I POST a slide entity with 4 analyses in it, i.e. do that in one request?